apps/condo/domains/billing/constants.js

```diff
@@ -13,10 +13,12 @@ const BILLING_INTEGRATION_ORGANIZATION_CONTEXT_STATUSES = [
     BILLING_INTEGRATION_ORGANIZATION_CONTEXT_FINISHED_STATUS,
 ]
 // End of danger-change zone
+const BILLING_INTEGRATION_DEFAULT_PAGE_TITLE = 'Биллинг'
```

Comment:
add translations

Reply:
resolved by setting billingPageTitle back to non-required, and get default translated value from client